test/Android/Resources/layout/Main.axml

```diff
@@ -15,7 +15,7 @@
         android:text="@string/Logout" />
     <TextView
         android:id="@+id/UserDetailsTextView"
-        android:text="Text"
+		android:scrollbars="vertical"
```

Comment:
is this enough to fix the scrolling issue?

Reply:
It is.